server/index.js

```diff
@@ -48,14 +48,32 @@ async function setupApp() {
 
   });
 
+  app.post('/users', async (req, res) => {
+    try {
+      const { name, email, password, birthdate, notification_preference, biography, terms_agreed, newsletter_subscribed } = req.body;
+      const user = await pgClient.query(`SELECT * FROM  messages WHERE email = $1`, [email]);
```

Comment:
Did you mean to check the users table?
```suggestion
      const user = await pgClient.query(`SELECT * FROM  users WHERE email = $1`, [email]);
```